modules/selection/src/main/kotlin/com.r3.corda.lib.tokens.selection/internal/ExternalIdIndexUtils.kt

```diff
@@ -0,0 +1,75 @@
+package com.r3.corda.lib.tokens.selection.internal
+
+import net.corda.core.crypto.toStringShort
+import net.corda.core.node.ServiceHub
+import net.corda.nodeapi.internal.persistence.CordaPersistence
+import java.security.PublicKey
+import java.util.*
+
+/** Table names. */
+val publicKeyHashToExternalId = "pk_hash_to_ext_id_map"
+
+/** Column names. */
+val publicKeyHashToExternalId_externalId = "external_id"
+val publicKeyHashToExternalId_publicKeyHash = "public_key_hash"
+
+sealed class Holder {
```

Comment:
Again let's revisit the design of this before we release.

Reply:
it's internal for indexing